package.json

```diff
@@ -32,6 +32,7 @@
     "import"
   ],
   "dependencies": {
+    "find-babel-config": "^1.2.0",
```

Comment:
Funny how I was thinking to deprecate the library as there should be a better "official" way of fetching the config now (see https://github.com/babel/babel/pull/7472)